apps/codelab/src/app/components/slides/title-slide/title-slide.component.ts

```diff
@@ -9,4 +12,36 @@ export class TitleSlideComponent {
   @Input() title: string;
   @Input() description: string;
   @Input() prereqs: string;
+
+  constructor(
+    private readonly router: Router,
+    private readonly presentation: SlidesDeckComponent,
+    @Optional() @Inject(MENU_ROUTES) readonly menuRoutes
+  ) {
+    if (this.presentation != null) {
```

Comment:
In which case would the presentation be null?

Reply:
since we removed @optional presentation will not be null... will remove the check...

Reply:
Drop the check?